mmcv/parallel/scatter_gather.py

```diff
@@ -43,17 +47,22 @@ def scatter_map(obj):
     try:
         return scatter_map(inputs)
     finally:
-        scatter_map = None
+        scatter_map = None  # type: ignore
 
 
-def scatter_kwargs(inputs, kwargs, target_gpus, dim=0):
+def scatter_kwargs(inputs: ScatterInputs,
+                   kwargs: ScatterInputs,
+                   target_gpus: list,
+                   dim: int = 0) -> Tuple[tuple, tuple]:
     """Scatter with support for kwargs dictionary."""
     inputs = scatter(inputs, target_gpus, dim) if inputs else []
     kwargs = scatter(kwargs, target_gpus, dim) if kwargs else []
     if len(inputs) < len(kwargs):
-        inputs.extend([() for _ in range(len(kwargs) - len(inputs))])
+        inputs.extend([() for _ in range(len(kwargs) - len(inputs))
+                       ])  # type: ignore
```

Comment:
Can we use 
`inputs.extend([() for _ in range(len(kwargs) - len(inputs))])  # type: ignore  # noqa: E501`
instead?